src/plots/cartesian/axes.js

```diff
@@ -944,9 +952,61 @@ axes.calcTicks = function calcTicks(ax, opts) {
             axes.prepTicks(mockAx, opts);
         }
 
+        if(mockAx.tickmode === 'full domain') {
+            var nt = mockAx.nticks;  // does mockAx have nitkcs?
```

Comment:
```suggestion
            var nt = mockAx.nticks;
```
Yep, you ensured that in [tick_value_defaults](https://github.com/plotly/plotly.js/pull/6827/files#diff-a9bf8cdd875c89f6be26c1fc2187f146322c4f7150cd6602305808ea1930a5ebR32), then (for minor ticks) this gets pulled into `mockAx` by the `extendFlat`.